### PR TITLE
GEODE-852: refactor gemfire-pulse not to generate a pulseverion.prope…

### PR DESCRIPTION
**Pull request by jinmeiliao**
…rties file but copy it from gemfire-core

### CODE REVIEW

File: gemfire-pulse/src/main/java/com/vmware/gemfire/tools/pulse/internal/data/PulseConstants.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
   // Pulse version details properties from properties file
   public static final String PULSE_VERSION_PROPERTIES_FILE = "pulseversion.properties";
```
Comment (nitin-amp) [inline gemfire-pulse/src/main/java/com/vmware/gemfire/tools/pulse/internal/data/PulseConstants.java]: Should this String change to 'GemFireVersion.properties'?

**Comment by jinmeiliao** [comment]
To avoid confusion, do not rename the properties file copied from gemfire-core

**Comment by upthewaterspout** [comment]
Looks good! One minor point - I think this is missing something that will tell gradle what the copyPulsePropFile  task depends on. You could either add a dependsOn to the task, or maybe add an inputs.dir to the task:

inputs.dir project(':gemfire-core').sourceSets.main.output

Try running 
/gradlew  gemfire-pulse:copyPulsePropFile
or
/gradlew  gemfire-pulse:war

and you'll see what I mean; it won't generate the original properties file.

**Comment by jinmeiliao** [comment]
Good point. Will create another pull request for this

**Pull request closed without merging**